src/reactions/github/fork.js

```diff
@@ -0,0 +1,21 @@
+class Fork {
+	constructor(streamlabs) {
+		this.streamlabs = streamlabs;
+	}
+
+	async handle(event, payload) {
```

Comment:
Do we need the `event`? Also, I prefer this as a single object param with destructuring :D